### ISSUE TITLE
about the datasets

### ISSUE THREAD

**Issue by WangzcBruce** [FreeformRobotics/Progressive-Self-Distillation-for-Ground-to-Aerial-Perception-Knowledge-Transfer#1]
Is there any other links of downloading the datasets? Because of the firewall, it's hard to download a huge dataset from thegoogle netdisk. Thank you anyway!

**Comment by JunjH** [comment]
Please download via https://pan.baidu.com/s/1i6zE0HBDsl4_4GYff6AKnQ?pwd=1111

**Comment by 1nanyihao** [comment]
The downloaded training code is missing the loss.py file and img_transforms.py file. I have been debugging the code recently but with little success. I am writing to you boldly to ask if you could kindly share the complete training code for this paper?
